### PR TITLE
fix(use-controlled): value doest't change when controlledValue changed

### PR DESCRIPTION
**Pull request by superman66**

### CODE REVIEW

**Comment by SevenOutman** [comment]
缺少了验证此变更的测试用例

**Review by SevenOutman — CHANGES_REQUESTED** [review]
缺少了验证此变更的测试用例

File: package-lock.json [hunk 1]
```diff
@@ -1926,6 +1926,19 @@
         "@testing-library/dom": "^7.28.1"
       }
     },
+    "@testing-library/react-hooks": {
+      "version": "7.0.0",
+      "resolved": "https://npm.hypers.cc/@testing-library%2freact-hooks/-/react-hooks-7.0.0.tgz",
```
Comment (simonguo) [inline package-lock.json:1931]: registry 切换到 npm: registry.npmjs.org

**Review by SevenOutman — APPROVED** [review]
LGTM

**Pull request merged**